src/core/AutomationPattern.cpp

```diff
@@ -597,11 +600,14 @@ void AutomationPattern::loadSettings( const QDomElement & _this )
 		}
 	}
 	
-	if( _this.hasAttribute( "stylecolor" ) )
-	{
-		setUseStyleColor( _this.attribute( "stylecolor" ).toInt() );
-		setUseCustomClipColor( _this.attribute( "clipcolor" ).toInt() );
-		setColor( _this.attribute( "color" ).toUInt() );
+	if( _this.hasAttribute( "color" ) )
+	{
+		useStyleColor( false );
+		useCustomClipColor( _this.attribute( "isclipcolor" ).toInt() );
+		
+		QColor loadedColor;
+		loadedColor.setNamedColor( _this.attribute( "color" ) );
+		setColor( loadedColor );
```

Comment:
You can use [this constructor](https://doc.qt.io/qt-5/qcolor.html#QColor-5) directly.
```suggestion
		setColor( _this.attribute( "color" ) );
```

Reply:
Oh, didn't know it exists. Will update